lib/commands/backup.js

```diff
@@ -0,0 +1,178 @@
+'use strict';
+const Command = require('../command');
+const Promise = require('bluebird');
+const createDebug = require('debug');
+const Zip = require('adm-zip');
+const walker = require('klaw-sync');
+const fs = require('fs-extra');
+const path = require('path');
+const cli = require('../');
+
+const debug = createDebug('ghost-cli:backup');
+
+function absolutePath(inputPath) {
+    return path.isAbsolute(inputPath) ? inputPath : path.join(process.cwd(), inputPath);
+}
+
+class BackupCommand extends Command {
+    run(argv) {
+        return this.ui.listr([{task: this.initialize.bind(this),
```

Comment:
Because I'm a bit OCD 😁  (and it makes it a bit easier to read) can you split this out into multiple lines, e.g.
```js
return this.ui.listr([{
  task: this.initialize.bind(this),
  title: 'Checking Configuration'
}], {argv: argv}).then(() => {
  // omitted code
});
```